.lintrunner.toml

```diff
@@ -653,3 +653,31 @@ init_command = [
     '--dry-run={{DRYRUN}}',
     'PyYAML==6.0',
 ]
+
+# This also provides the same functionality as BLACK formatter. Just to be
+# on the safe side, we will run both BLACK and UFMT for a while to make sure
+# that nothing breaks before removing the former
+[[linter]]
+code = 'UFMT'
+# This is a big bang change to reformat every python files in the repo,
+# so it's arguably safer to gradually reformat them in subsequent PR.
+# After that, we can use **/*.py to apply the formatter to everything
+include_patterns = ['tools/linter/**/*.py']
```

Comment:
My strategy here is to slowly re-formatting the rest of the repo once this PR is merged. Here is a glimpse of all the changes looks like https://paste.sh/fX86Cq5p#yVyig1IAqkD9GlplM_qnWG0f. I see:

* Sorted imports
* Single quotes to double quotes
* Code style